client/views/editor/editor.html

```diff
@@ -1,3 +1,9 @@
 <template name="Editor">
-    {{> CodeMirror id="some-id" name="someName" options=editorOptions code=editorCode reactiveVar="varName"}}
+    {{> CodeMirror id="id_codemirror" name="name_codemirror" options=editorOptions code=editorCode reactiveVar="varName"}}
```

Comment:
the normal convention for id, name and class name are with - (dashes) instead of _ (underscores)